Policies/github-bug-bounty-program-legal-safe-harbor.md

```diff
@@ -31,4 +31,4 @@ That said, if legal action is initiated by a third party, including law enforcem
 
 ### 3. Limited Waiver of Other Site Polices
```

Comment:
- @Daniel-Mietchen I'm really trying idk what I'm doing Idk how I even set this coding thing up awhile back.

Reply:
Looks good